packages/block-editor/src/components/list-view/block.js

```diff
@@ -67,8 +67,15 @@ function ListViewBlock( {
 	const { toggleBlockHighlight } = useDispatch( blockEditorStore );
 
 	const blockInformation = useBlockDisplayInformation( clientId );
-	const blockName = useSelect(
-		( select ) => select( blockEditorStore ).getBlockName( clientId ),
+	const { isContentLocked, blockName } = useSelect(
```

Comment:
Small PR to get the locking data all in a single hook at https://github.com/WordPress/gutenberg/pull/43803